### DISCUSSION TITLE
Share pictures of where you run!

### DISCUSSION THREAD

**Discussion by ColinFay** [ColinFay/ur-first-5k#65] [Show and tell]
Hey all! 

You're 16 now doing this challenge, and it seems that we are all in different parts of the world 🥳

I just thought it would be nice to share here some pictures of where you run, so that we can (figuratively) travel together ✈

Cheers, 
Colin

**Comment by Felixmil** [comment]
I run in a park 10 min from home in Saint-Etienne, France.
![image](https://user-images.githubusercontent.com/34234913/104588841-4428a000-5669-11eb-8a1c-a5640c170bdc.jpeg)
This park was previously an iron mine (until 1973). There's still the main shaft !
![image](https://user-images.githubusercontent.com/34234913/104588860-4be84480-5669-11eb-8745-81586b62303a.jpeg)

**Reply by ColinFay** [reply]
That's an awesome view. I love the old industrial buildings like these :)

**Reply by Felixmil** [reply]
Yeah me too, particularly when nature takes over, it creates a very unique atmosphere.

**Comment by jdtrat** [comment]
That's super cool @Felixmil ! Here's the view from the park next to my house in Orlando, Florida! It doesn't have an iron mine, but it was a golf course until a few years ago😄
![Screen Shot 2021-01-19 at 8 37 50 PM](https://user-images.githubusercontent.com/51417262/105115092-34102680-5a96-11eb-8327-c7339ac6a0c0.png)

**Comment by fazepher** [comment]
I usually run around campus, at the University of Warwick. 
Yesterday I only walked along my usual routes because it was snowing ❄️❄️☃️🌨⛄️😃😄 
But I found people that were running or cycling. Here are some pictures: 

![image](https://user-images.githubusercontent.com/24779456/105746900-f39e2600-5f37-11eb-8331-3f615f9aa039.png)
![image](https://user-images.githubusercontent.com/24779456/105746931-0153ab80-5f38-11eb-82a0-34795bad50d7.png)
![image](https://user-images.githubusercontent.com/24779456/105746945-087ab980-5f38-11eb-8c45-f728f450dde8.png)
![image](https://user-images.githubusercontent.com/24779456/105746968-0f093100-5f38-11eb-8c45-655c6bda3355.png)
![image](https://user-images.githubusercontent.com/24779456/105747006-1a5c5c80-5f38-11eb-8e5e-cca05dc3397b.png)
![image](https://user-images.githubusercontent.com/24779456/105747148-4f68af00-5f38-11eb-9c9a-2d6d227b7d01.png)

**Reply by jdtrat** [reply]
That looks beautiful! I'm heading back to my college soon and will send some pictures once I go running around campus.

**Comment by ColinFay** [comment]
Anybody else got snow this week ?

![E2745039-3FC5-4062-9832-460EF999C990](https://user-images.githubusercontent.com/17936236/107534873-88538580-6bc0-11eb-91da-7b487c525234.jpeg)
![A8ED0702-EC9D-4213-B93D-AFC3C9E02D97](https://user-images.githubusercontent.com/17936236/107534891-8d183980-6bc0-11eb-9f72-e0d625323012.jpeg)
![25797EB1-34F9-4820-A405-F70F2DEF771C](https://user-images.githubusercontent.com/17936236/107534900-91445700-6bc0-11eb-9446-bde9a5e8e051.jpeg)
![844478E3-A091-4224-BF44-953427B2DA18](https://user-images.githubusercontent.com/17936236/107534909-91dced80-6bc0-11eb-85df-b4e37d9f48cf.jpeg)
![0593D556-64B7-497D-9EF9-37A042FFDED2](https://user-images.githubusercontent.com/17936236/107534916-930e1a80-6bc0-11eb-8347-4729f866d19c.jpeg)
![3420AAE6-2E83-4332-8894-54BED3B1308A](https://user-images.githubusercontent.com/17936236/107534921-94d7de00-6bc0-11eb-9f62-03048a8d12bf.jpeg)
![D9FD6D1B-B854-4E4C-8EDE-94220E1B14B2](https://user-images.githubusercontent.com/17936236/107534924-96090b00-6bc0-11eb-8090-6497b930cc49.jpeg)
